gdcdictionary/schemas/copy_number_auxiliary_file.yaml

```diff
@@ -0,0 +1,86 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "copy_number_auxiliary_file"
+title: Copy Number Auxiliary File
+type: object
+namespace: https://gdc.cancer.gov
+category: data_file
+project: '*'
+program: '*'
+description: >-
+  Data file related to the copy number pipeline that contains any outputs not strictly defined in other nodes
+additionalProperties: false
+submittable: false
+downloadable: true
+previous_version_downloadable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - file_state
+  - state
+  - error_type
+
+links:
+  - name: somatic_copy_number_workflows
+    backref: copy_number_auxiliary_files
+    label: derived_from
+    target_type: somatic_copy_number_workflow
+    multiplicity: one_to_one
+    required: true
+
+required:
+  - submitter_id
+  - file_name
+  - file_size
+  - md5sum
+  - data_category
+  - data_format
+  - data_type
+  - experimental_strategy
+  - platform
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+properties:
+
+  $ref:
+    - "_definitions.yaml#/data_file_properties"
+
+  data_category:
+    $ref:
+      - "_terms.yaml#/data_category/common"
+    enum:
+      - Copy Number Variation
+
+  data_type:
+    $ref:
+      - "_terms.yaml#/data_type/common"
+    enum:
+      - Intermediate Analysis Archive
+
+  data_format:
+    $ref:
+      - "_terms.yaml#/data_format/common"
+    enum:
+      - TAR
+
+  experimental_strategy:
+    $ref:
+      - "_terms.yaml#/experimental_strategy/common"
+    enum:
+      - WGS
```

Comment:
ticket says WXS

Reply:
and do we need the enumDef portion for exp_strat? (I'm not familiar with when we should have it and when not)

Reply:
Changed JIRA.   Also, for enumDef, I usually keep whatever is there for other nodes.  We have a ref somewhere for experimental_strategy, let's point to it.